helper/build.gradle.kts

```diff
@@ -15,7 +15,7 @@ android {
 
     buildTypes {
         named("release") {
-            isMinifyEnabled = true
+            isMinifyEnabled = false
```

Comment:
We can still minify the library to reduce size. What we need to fix is to write the correct proguard.